contributors/triage/README.md

```diff
@@ -0,0 +1,13 @@
+# Kubernetes Issue Triage
```

Comment:
@bgrant0607 hi Brian, agree but lot of content besides labels can be used as a guidelines for other repos. I can try adding a note with some of the text from your comment and we can keep the scope of guidelines more than just kubernetes repo?  As you mentioned other repos don't have a real problem of issue management because of low volume of issues, but I heard about some more code splitting, for example, new repos like kubernetes/kubectl are under progress and I guess such repos will be similar to kubernetes/kubernetes repo. Thanks!